src/element-overlay.ts

```diff
@@ -1,23 +1,23 @@
-import { BoundingBox } from "./utils";
+import { BoundingBox, ElementOverlayOptions } from "./utils";
 
 export default class ElementOverlay {
   overlay: HTMLDivElement;
   shadowContainer: HTMLDivElement;
   shadowRoot: ShadowRoot;
   usingShadowDOM?: boolean;
 
-  constructor() {
+  constructor(options: ElementOverlayOptions) {
     this.overlay = document.createElement("div");
-    this.overlay.className = "_ext-element-overlay";
-    this.overlay.style.background = "rgba(250, 240, 202, 0.2)";
-    this.overlay.style.borderColor = "#F95738";
-    this.overlay.style.borderStyle = "solid";
-    this.overlay.style.borderRadius = "1px";
-    this.overlay.style.borderWidth = "1px";
-    this.overlay.style.boxSizing = "border-box";
-    this.overlay.style.cursor = "crosshair";
-    this.overlay.style.position = "absolute";
-    this.overlay.style.zIndex = "2147483647";
+    this.overlay.className = options.className || "_ext-element-overlay";
+    this.overlay.style.background = options.background || "rgba(250, 240, 202, 0.2)";
```

Comment:
I think all these lines need to use `options.style.{property}` (although you might want to extract `style` to its own variable to above). It looks like is currently causing the [build to fail](https://github.com/hmarr/pick-dom-element/runs/7119295345?check_suite_focus=true#step:4:17).